packages/attribute-slicer/src/AttributeSlicer.ts

```diff
@@ -125,6 +125,23 @@ export class AttributeSlicer {
     private searchDebounce = SEARCH_DEBOUNCE;
 
     /**
+     * Whether or not to left align item text
+     */
+    private _leftAlignText = false;
+    get leftAlignText() {
+        return this._leftAlignText;
+    }
+    
+    /**
+     * Sets wheter or not to left align item text
+     */
+    set leftAlignText(value: boolean){
+        if (value !== this._leftAlignText){
+            this._leftAlignText = value;
```

Comment:
Does that mean we would be rerendering the list twice in the option in the formatting pain is used or is there a way we can avoid the rerender if its already being done?